commonTools/framework/CMakeLists.txt

```diff
@@ -125,8 +125,8 @@ create_get_changed_trilinos_packages_test(ProjectsList_TeuchosCore
 
 create_get_changed_trilinos_packages_test(cmake_std_atdm_TeuchosCore_PyTrilinos
   "cmake/std/atdm/anything\npackages/teuchos/core/CMakeLists.txt\npackages/PyTrilinos/anything"
-  "TeuchosCore,PyTrilinos"
-  "TeuchosCore"
+  "TrilinosATDMConfigTests,TeuchosCore,PyTrilinos"
+  "TrilinosATDMConfigTests,TeuchosCore"
```

Comment:
Should this be: `"ATDMTrilinosConfigTests,TeuchosCore"`?

Reply:
No, I changed the package name to `TrilinosATDMConfigTests` to be consistent with the existing package `TrilinosFrameworkTests`.

Reply:
I just changed the name `ATDMTrilinosConfigTest` to `TrilinosATDMConfigTest` in the above description to avoid confusion.  Sorry about the confusion.